Makefile

```diff
@@ -297,6 +297,9 @@ target/%/vector.tar.gz: target/%/vector CARGO_HANDLES_FRESHNESS
 .PHONY: test
 test: ## Run the unit test suite
 	${MAYBE_ENVIRONMENT_EXEC} cargo nextest run --workspace --no-fail-fast --no-default-features --features "${DEFAULT_FEATURES}" ${SCOPE}
+ifneq ($(OPERATING_SYSTEM), Windows)
+	@scripts/upload-test-results.sh
+endif
```

Comment:
Might be worth leaving a comment about why we don't run this on Windows for future readers.

Reply:
Fair - more usable than just the github issue,